luigi/configuration/core.py

```diff
@@ -66,9 +66,26 @@ def add_config_path(path):
     # select parser by file extension
     _base, ext = os.path.splitext(path)
     if ext and ext[1:] in PARSERS:
-        parser_class = PARSERS[ext[1:]]
+        parser = ext[1:]
     else:
-        parser_class = PARSERS[PARSER]
+        parser = PARSER
+    parser_class = PARSERS[parser]
+
+    if not parser_class.enabled:
+        msg = (
+                "Parser not installed yet. "
+                "Please, install luigi with required parser:\n"
+                "pip install luigi[{parser}]"
+            )
+        raise ImportError(msg.format(parser=parser))
```

Comment:
This `if not parser_class.enabled` is checked at least twice here. Can that be extracted into a method or something for DRY programming purposes?

Reply:
Fixed